src/mon/MonmapMonitor.h

```diff
@@ -83,6 +84,26 @@ class MonmapMonitor : public PaxosService {
 
  private:
   bufferlist monmap_bl;
+
+  class C_ApplyFeatures : public Context {
```

Comment:
as this class is not used elsewhere, can we define it in MonmapMonitor.cc at the place where it is used, so we are able to find it's implementation right where it is used?